### DISCUSSION TITLE
Como traducir al español esta aplicación

### DISCUSSION THREAD

**Discussion by YoandrisColl** [jeffvli/sonixd#215] [Q&A]
Hola como pudiera traducir esta app al español

**Comment by jeffvli** [comment]
Hi, I may add detailed instructions for translation soon, but the general instructions for now is:
1. Fork the Sonixd repository
2. Open [src/i18n/i18next-parser.config.js](https://github.com/jeffvli/sonixd/blob/main/src/i18n/i18next-parser.config.js) and add your language code to the `locales` array. (For Spanish this would look like `locales: ['en', 'de', 'zhcn', 'es'],`)
3. Either run `yarn i18next` to automatically create the new locale file, or copy [src/i18n/locales/en.json](https://github.com/jeffvli/sonixd/blob/main/src/i18n/locales/en.json) to a new file in the same folder named after the new locale. (For Spanish this would be `src/i18n/locales/es.json`)
4. Translate all the words inside the new file and save it.
5. Commit and push the changes to your forked repository, and then create a pull request.

You can look at this pull request as an example #191
